test/functional/test_runner.py

```diff
@@ -126,6 +126,7 @@
     'p2p-fingerprint.py',
     'uacomment.py',
     'p2p-acceptblock.py',
+    'loggingtest.py',
```

Comment:
Consider naming the file "function_debuglogfile.py" to match the naming scheme in #11796 ?

Reply:
`function_` isn't one of the allowed prefixes according to #11796?

Also I think it would make sense to extend this test later to test other logging related things such as the `logging` RPC call, that's why I used a more general name than "debuglogfile".

Reply:
Gah. feature_. Apparently there's more that one two-syllable word beginning with f.

Reply:
ok renamed to simply `feature_logging.py`